### DISCUSSION TITLE
posts no longer viewable in archive after mass privating

### DISCUSSION THREAD

**Discussion by nymphpassing** [AprilSylph/XKit-Rewritten#1379] [Help Desk] [answered]
i used the mass privater feature for one month as a test and it worked, but i can no longer see the posts in my archive like i would if i'd done it manually through tumblr's mass editor feature, is this an error or how the feature operates with this extension?

**Comment by marcustyphoon** [comment]
Mass Privater simulates using the mass post editor (it looks the same to Tumblr's backend), so this could definitely be a weird bug but it seems unlikely. If you private some posts using the actual mass post editor, do you see the same thing happening? Could be a Tumblr issue.

**Reply by nymphpassing** [reply]
thank you for getting back to me so quickly! no, when i mass private posts using tumblr's tool, i can still see them in my archive with a lock to indicate they're now private

**Reply by marcustyphoon** [reply, marked as answer]
Hm—I tried one post one way and one the other way, and I see both of them with a lock icon on `https://www.tumblr.com/mega-editor/published/[myblog]` and see neither of them on `https://[myblog].tumblr.com/archive`—which do you mean when you say "in my archive"?

(I do remember someone explaining to me that the archive does a weird thing with at least one kind of post, but I don't recall the details so I'm not immediately sure if that's related.)

**Reply by nymphpassing** [reply]
ah this is my mistake! i thought that when you privated posts using either tool they were still visible in your archive - i'm seeing now they're only visible, even as private through mass post editor, as i'm new to both features! sorry for taking up your time, thank you for looking into this so promptly and your work to help maintain this extension!
